### PR TITLE
Release Info

### PR DESCRIPTION
**Pull request by josephkane**
* allow users to acces information about deployed apps from cluster details page

* fix for https://github.com/supergiant/control/issues/957

### CODE REVIEW

File: cmd/ui/assets/src/app/app.module.ts [hunk 1]
```diff
@@ -317,7 +319,8 @@ import { KubectlConfigModalComponent } from './clusters/cluster/kubectl-config-m
     DeleteClusterModalComponent,
     DeleteReleaseModalComponent,
     SshCommandsModalComponent,
-    KubectlConfigModalComponent
+    KubectlConfigModalComponent,
+    ReleaseInfoModalComponent
```
Comment (nikage) [inline cmd/ui/assets/src/app/app.module.ts]: why the components are included into import section?

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss [hunk 2]
```diff
@@ -755,6 +755,20 @@
     }
   }
 
+  .info-icon {
+    background-image: url(/assets/img/info-pink.svg);
```
Comment (nikage) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: this won't work with aot build, I think inital slash should be removed. needs to be checked out with `npm run build:prod`

File: cmd/ui/assets/src/app/app.module.ts (hunk 1 above)
Comment (josephkane) [reply cmd/ui/assets/src/app/app.module.ts]: modals are included here so they are loaded dynamically

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss (hunk 2 above)
Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: thanks, will check and fix file-wide if it is an issue

Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: @nikage I wasn't able to get this to break, it seems to work fine. would you mind fetching this PR and testing it?

File: cmd/ui/assets/src/app/app.module.ts (hunk 1 above)
Comment (nikage) [reply cmd/ui/assets/src/app/app.module.ts]: yeap, sorry. Github line collapsing confused me. Then that is fine

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss (hunk 2 above)
Comment (nikage) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: this won't break the build process itself, but the image link might be broken. I'll check this in a while

Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: thats what I mean, I couldn't get the image link to break. I might be doing something wrong though

**Comment by eldridgeh** [comment]
This looks to be working. However, was it intended to split the cluster name on the inline machine info? This probably isn't a big deal, but I don't think we do this anywhere else anymore
screen shot 2018-11-30 at 11 58 05 am
.

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss (hunk 2 above)
Comment (nikage) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: yeap there is an issue. Here is the icon on localhost:4200 e.g dev server
image

Comment (nikage) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: and here is localhost:8080 e.g. build 
image

Comment (nikage) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.scss]: apparently, no. it will be okay.  that was a browser cache issue, not the code

**Pull request merged**